src/main.py

```diff
@@ -344,12 +361,20 @@ def projectSingleCashflow(cf, start, end, life, lifeCf, taxMult, inflRate, proje
     @ In, inflRate, float, inflation rate multiplier (1 - inflation)
     @ In, projectLength, int, total years of analysis
     @ In, v, int, verbosity
+    @ In, pyomoSing, pyo.ConcreteModel(), concrete model including all variables
```

Comment:
as with the others, can we get by with a boolean check here? also optional tag in docstring